.github/workflows/continuous-integration.yml

```diff
@@ -73,12 +73,22 @@ jobs:
         php-version:
           - "7.4"
           - "8.1"
+        oracle-version:
+          - "11"
+          - "latest"
```

Comment:
Let's use a fixed latest version as of now, e.g. 21. We will add the support for new versions explicitly.

Reply:
Done